docusaurus/docs/troubleshooting.md

```diff
@@ -83,3 +83,11 @@ module.exports = {
   transformIgnorePatterns: [nodeModulesToTransform([...grafanaESModules, 'packageName'])],
 };
 ```
+
+### I get `"image with reference <plugin-name> was found but does not match the specified platform: wanted linux/amd64, actual: linux/arm64/v8"` after running `docker-compose up` or `npm run server`.
+
+This error is most likely to impact users of Mac computers with Apple silicon. If you have previously built an image for a plugin scaffolded with `create-plugin` prior to v1.12.2, then running `docker-compose up` may fail with the above message if the old image hasn't been removed.
+
+#### What you can do:
+- Remove the image using `docker rmi <plugin-name> --force`
```

Comment:
Should we add a preliminary step here for `docker-compose down` to stop and remove the container first? It should mean there is no need to use `--force` flag in `docker rmi` command or the `--build` flag in `docker-compose up` command?

Reply:
fair point - will update